server/src/main/java/org/elasticsearch/http/netty4/Netty4HttpServerTransport.java

```diff
@@ -546,6 +547,7 @@ public void initChannel(Channel ch) throws Exception {
             ch.pipeline().addLast("encoder", new HttpResponseEncoder());
             final HttpObjectAggregator aggregator = new HttpObjectAggregator(Math.toIntExact(transport.maxContentLength.getBytes()));
             aggregator.setMaxCumulationBufferComponents(transport.maxCompositeBufferComponents);
+            ch.pipeline().addLast("chunked", new ChunkedWriteHandler());
```

Comment:
HTTP uses `DefaultFileRegion` for zero copy instead of manual chunking so it doesn't depend on the ChunkedWriteHandler. Zero copy doesn't work via HTTPS